src/components/Tabs/Tab/Tab.scss

```diff
@@ -57,7 +57,7 @@
   }
 
   &.tab-focus-visible-inset {
-    @include focus-style-inset(3px, 3px);
+    @include focus-style-css-inset(3px, 3px);
```

Comment:
the other mixin depend on using focus-visible class name which had different purpose (we inject it when we do real focus for old browsers support)